far/farlang.templ.m4

```diff
@@ -7681,6 +7681,21 @@ upd:"Occurrences: {0}, lines: {1}"
 upd:"Occurrences: {0}, lines: {1}"
 upd:"Occurrences: {0}, lines: {1}"
 
+MEditSearchAllFileNameAppend
```

Comment:
Do we need this?
I doubt that anyone will ever want to change it, and changing it can do more harm than good anyway.

Reply:
Do you mean to hard code the string "Filtered" instead? I can do that. Yeah, I thought somebody may want to translate it to their language. Out of curiosity, what harm can come out of it?

I'd rather keep it but I am not strong on it. Please let me know one way or the other.

Reply:
Someone could put unsupported characters there and then wonder why it doesn't work.
On the other hand, we do have MNewFileName there already, so I won't insist.

Reply:
Good point. However, the name is used only in the "Save As" dialog, so one will immediately see what one is about to do. 😀

I'll keep it for now.